crates/fortitude_linter/src/rules/correctness/implicit_typing.rs

```diff
@@ -89,66 +89,6 @@ impl AstRule for InterfaceImplicitTyping {
     }
 }
 
-/// ## What it does
-/// Checks for unnecessary `implicit none` in module procedures
-///
-/// ## Why is this bad?
-/// If a module has 'implicit none' set, it is not necessary to set it in contained
-/// functions and subroutines (except when using interfaces).
-#[derive(ViolationMetadata)]
-pub(crate) struct SuperfluousImplicitNone {
```

Comment:
Had we moved this or something?

Reply:
It was in the same file as the other `implicit none` rules, so I think it's just been lurking there since before we overhauled the rules categories. No idea how the linters missed it until now.